project.clj

```diff
@@ -31,7 +31,7 @@
   :lein-release {:scm :git
                  :deploy-via :lein-deploy}
 
-  :parent-project {:coords [puppetlabs/clj-parent "4.9.1"]
+  :parent-project {:coords [puppetlabs/clj-parent "5.3.1"]
```

Comment:
```suggestion
  :parent-project {:coords [puppetlabs/clj-parent "5.3.0"]
```
We should pin the hikaricp version in this project until we can complete the clj-parent dance.  I think that would be:
release this with parent at 5.3.0 and hikcariCP pinned.  Update clj-parent with the new release of this and hikariCP.  Update this again to remove the hikari pin, and release again.

Reply:
Changed parent to 5.3.0 and pinned Hikari here, I see the dance you mean now